packages/react/vite.config.ts

```diff
@@ -61,8 +69,7 @@ export default defineConfig(({ mode }) => {
 
                 if (file?.type === "chunk") {
                   // Replace .css imports and requires
-                  const pattern =
```

Comment:
We need to keep the `require` portion of the regex as that strips the unnecessary `.css.js` imports from the cjs build. For example, if you build and look at `react/dist/cjs/modules/core/components/Button/Button.js` you'll see it includes `require("./styles.css.js");`

`/(import\s+['"].+\.css\..*?['"];?)|(require\(['"][^()]+\.css(\.js)?['"]\);?)/g`